features/lwipstack/LWIPInterface.cpp

```diff
@@ -267,6 +272,12 @@ char *LWIP::Interface::get_mac_address(char *buf, nsapi_size_t buflen)
     return buf;
 }
 
+char *LWIP::Interface::get_interface_name(char *buf)
+{
+    sprintf(buf, "%c%c%d", netif.name[0], netif.name[1], netif.num);
```

Comment:
I think there should be e.g. a buffer length field as an parameter to ensure that sprintf does not write over the allocated buffer length.

Reply:
Otherwise looks good to me.